packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { UploadTask } from '@aws-amplify/storage';
+import { DragActionHandlers } from './hooks/useFileUploader/types';
+import { ButtonProps } from 'src/primitives';
+
+export type SetShowPreviewer = (show: boolean) => void;
+type LevelInfo = 'public' | 'protected' | 'private';
+export type Files = File[];
+
+export interface UploadButtonProps extends ButtonProps {
+  className?: string;
+  disabled?: boolean;
+  onClick?: () => void;
```

Comment:
Maybe I'm wrong but aren't these props just available on `ButtonProps`?

Reply:
I'll delete it, and I'll change to `isDisabled` to match ButtonProps

Reply:
[d23a487](https://github.com/aws-amplify/amplify-ui/pull/3028/commits/d23a487464ab79fd1372e0165c1ab57616dde160)